crates/rattler_conda_types/src/version_spec/constraint.rs

```diff
@@ -143,27 +58,27 @@ mod test {
     fn test_invalid_op() {
         assert_eq!(
             Constraint::from_str("<>1.2.3"),
-            Err(ParseConstraintError::InvalidOperator)
+            Err(ParseConstraintError::InvalidOperator(String::from("<>")))
         );
         assert_eq!(
             Constraint::from_str("=!1.2.3"),
-            Err(ParseConstraintError::InvalidOperator)
+            Err(ParseConstraintError::InvalidOperator(String::from("=!")))
         );
         assert_eq!(
             Constraint::from_str("<!=1.2.3"),
-            Err(ParseConstraintError::InvalidOperator)
+            Err(ParseConstraintError::InvalidOperator(String::from("<!=")))
         );
         assert_eq!(
             Constraint::from_str("<!>1.2.3"),
-            Err(ParseConstraintError::InvalidOperator)
+            Err(ParseConstraintError::InvalidOperator(String::from("<!>")))
         );
         assert_eq!(
             Constraint::from_str("!=!1.2.3"),
-            Err(ParseConstraintError::InvalidOperator)
+            Err(ParseConstraintError::InvalidOperator(String::from("!=!")))
         );
         assert_eq!(
```

Comment:
Maybe we could add a test for `=>2.1` (that got translated by `conda-build` to `>2.1*`, but IMO we should error out.